tests/integration/relations/pgbouncer_provider/application-charm/charmcraft.yaml

```diff
@@ -5,12 +5,11 @@ type: charm
 bases:
   - build-on:
       - name: "ubuntu"
-        channel: "20.04"
+        channel: "22.04"
     run-on:
       - name: "ubuntu"
-        channel: "20.04"
-
+        channel: "22.04"
 
 parts:
   charm:
-    charm-binary-python-packages: [psycopg2-binary==2.9.5, setuptools]
\ No newline at end of file
+    charm-binary-python-packages: [psycopg2-binary==2.9.5, setuptools]
```

Comment:
I don't think we need `setuptools` here any more.

Reply:
Tnx, I have cherry-picked your commit.